tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -229,6 +231,7 @@ def test_hyperopt_executor_with_metric(use_split, csv_filename, tmpdir, ray_clus
     )
 
 
+@pytest.mark.slow
 @pytest.mark.distributed
 @pytest.mark.parametrize("backend", ["local", "ray"])
 def test_hyperopt_run_hyperopt(csv_filename, backend, tmpdir, ray_cluster_4cpu):
```

Comment:
I know hyperopt with Ray is not a P0 priority right now, but I do thin we should leave at least one of these tests in even if they're slow because otherwise we're going to get feedback from any failures on these tests post merge which somehow I don't feel too comfortable with given the hyperopt + ray + ludwig code is super specialized in finicky

Reply:
+1

Reply:
Sure thing. I've unmarked this test.